css/style.css

```diff
@@ -26,11 +24,150 @@ video {
 }
 
 .container {
-  margin: 0 auto;
+	position: relative;
+	margin: 0 auto;
   max-width: 960px;
   width: 100%;
+  height: 100%;
```

Comment:
This height should be set in a case per case basis instead of in the helper class, as per the discussion in this closed PR https://github.com/rtsc8/hostal_donde_las_paisas/pull/23, please reach out to me if you have any questions on how to make this change.

Reply:
Thank you. Done.